src/xref2/dune

```diff
@@ -7,39 +7,39 @@
  unix)
  (modules_without_implementation errors)
  )
-(alias
- (name   runtest)
+(rule
+ (alias   runtest)
  (deps   (:x paths.md)
          (package odoc))
  (enabled_if (> %{ocaml_version} 4.08))
  (action (progn
-           (run ocaml-mdx test %{x})
+           (run ocaml-mdx-test %{x})
            (diff? %{x} %{x}.corrected)
 )))
-(alias
- (name   runtest)
+(rule
+ (alias   runtest)
  (deps   (:x test.md)
          (package odoc))
  (enabled_if (> %{ocaml_version} 4.08))
  (action (progn
-           (run ocaml-mdx test %{x})
+           (run ocaml-mdx-test %{x})
            (diff? %{x} %{x}.corrected)
 )))
-(alias
- (name   runtest)
+(rule
+ (alias   runtest)
  (deps   (:x notes.md)
          (package odoc))
  (enabled_if (> %{ocaml_version} 4.08))
  (action (progn
-           (run ocaml-mdx test %{x})
+           (run ocaml-mdx-test %{x})
            (diff? %{x} %{x}.corrected)
 )))
-(alias
- (name   runtest)
+(rule
+ (alias   runtest)
  (deps   (:x scratch.md)
          (package odoc))
  (enabled_if (> %{ocaml_version} 4.08))
  (action (progn
-           (run ocaml-mdx test %{x})
+           (run ocaml-mdx-test %{x})
```

Comment:
I wonder if there is a way to fix this in Mdx ? The problem is that the binary `ocaml-mdx` depends at runtime on the binary `ocaml-mdx-test`. Is here a way to declare this to Dune ?

Reply:
not yet - see ocaml/dune#3499